llvm/include/llvm/ADT/APInt.h

```diff
@@ -2193,6 +2193,18 @@ inline const APInt absdiff(const APInt &A, const APInt &B) {
   return A.uge(B) ? (A - B) : (B - A);
 }
 
+///.Compute the signed floor of the average of C1 and C2
```

Comment:
Space instead of dot after `///`

Reply:
It is the average that is signed or unsigned, not the floor. So the comment should be "Compute the floor of the signed average of C1 and C2". Similarly for the others.